### PR TITLE
Deploying version 0.37.2 to AWS

### PR DESCRIPTION
**Pull request by Jack4589**
Also updated .gitignore to ignore files associated with pycharm ide.

# Description

Updated .gitignore to ignore files associated with pycharm ide.

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Change status
- [x] Complete, but not tested (may need new tests)

# Checklist:
Everything in this checklist is required for each PR.  Please do not approve a PR that does not have all of these items.  

- [x] I have pulled from master before submitting this PR
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My code has been reviewed by at least one peer (the peer review to approve a PR counts.  The reviewer must download and test the code)
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts

### CODE REVIEW

**Review by jbehringer95 — APPROVED** [review]
Looks good!

File: app/main.py [hunk 1]
```diff
@@ -18,7 +18,7 @@
     title='DS API - Family Promise',
     description=description,
     docs_url='/',
-    version='0.37.1',
+    version='0.37.2',
```
Comment (santorap) [inline app/main.py:21]: Nice job not adding a stray '/' like I did the first time I tried this!

File: .gitignore [hunk 2]
```diff
@@ -140,7 +140,11 @@ dmypy.json
 
 # Cython debug symbols
 cython_debug/
+
```
Comment (santorap) [inline .gitignore:143]: Thanks for separating out these sections

**Review by santorap — APPROVED** [review]
Looks good Jack!

**Pull request merged**